config/interpolate_funcs.go

```diff
@@ -487,11 +491,16 @@ func interpolationFuncDistinct() ast.Function {
 			var list []string
 
 			if len(args) != 1 {
-				return nil, fmt.Errorf("distinct() excepts only one argument.")
+				return nil, fmt.Errorf("accepts only one argument.")
```

Comment:
I learned that the caller prepends the function name automatically, so we were getting stutter in the resulting err messages.